index.html

```diff
@@ -1,207 +1,133 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Collins Akorda - Portfolio</title>
-  <link rel="stylesheet" href="style.css">
+  <link rel="Stylesheet" href="Style.css">
 </head>
-
 <body>
-
+  
   <header>
     <nav>
-      <div class="nav-bar">
-        <a href="#" class="logo">
-          <img class="menuIcone" src="./images/logo.png" alt="logo Image">
-        </a>
-
-        </button-->
-        <input type="checkbox" class="side-menu" id="side-menu" />
-        <label for="side-menu" class="hamb">
-          <span class="hamb-line"></span>
-        </label>
-        <div class="nav">
-          <ul class="menu">
-            <li><a href="#Portfolio">Portfolio</a></li>
-            <li><a href="#About">About</a></li>
-            <li><a href="#contact">Contacts</a></li>
-          </ul>
-        </div>
-
+		<div class="nav-bar">
+			<a href="#" class="logo">Collins Akorda </a>
+			<button class="hambug-menu">
+			<i class="menuIcon material-icons"> <img class="menuIcone" src="./Images/Union.png" alt="Union Image"></i>
+			<i class="closeIcon material-icons">X</i>
+			</button>
+			<ul class="mobile-menu">
+			  <li><a class="menuItem" href="#Portfolio">Portfolio</a></li>
+			  <li><a class="menuItem" href="#About">About</a></li>
+			  <li><a class="menuItem" href="#contact">Contacts</a></li>
+			</ul>
+	
+			<ul class="nav-links">
+			  <li class="links">Portfolio</li>
+			  <li class="links">About</li>
+			  <li class="links">Contact</li>
+			</ul>
+		  </div>
```

Comment:
- [ ] The changes you made to your image paths broke the images in the UI. Please you need to look into this as the images are not being displayed in the browser. :small_airplane: :wave:

Reply:
- [ ] As mentioned by the previous reviewer, the changes made your images path broke. Please notice that your image folder is  named `images` not `Images`, by changing that I think it will solve the broken images links issues.